data/src/main/java/org/akvo/flow/data/util/ApiUrls.java

```diff
@@ -31,6 +31,7 @@
     public static final String HMAC = "h";
     public static final String VERSION = "ver";
     public static final String ANDROID_ID = "androidId";
+    public static final String LAST_UPDATED = "lastUpdateTime";
```

Comment:
This will be removed in the next PR as it turns out we will not be using it anymore